### ISSUE TITLE
Launching "Animal Crossing: Let's go to the city" corrupts a majority of the cover images and banners. The titles are messed up too

### ISSUE THREAD

**Issue by TheMispeled** [Fledge68/WiiFlow_Lite#397]
**Describe the bug**
If you launch Animal crossing, it will return to the wii menu. Going back to wiiflow, a big portion of the cover images are corrupted. (a small portion at the top of the image is intact, the rest is colored static)

The titles are messed up too, for example Animal Crossing became "wii auto erase disc". Of course i didn't try to launch it again.

NFS Nitro became "nfs montreal" 

Super Mario galaxy 2 became " SUPER MARIO GALAXY MORE" 

Another NFS became "NFS11" and "Movie-ch install disc ver. A" ect...

**To Reproduce**
1. Launch Animal Crossing
2. You will be sent to the wii menu. Go back to wiiflow
3. See errors

**What version of WiiFlow Lite**
Version: 5.5.4

**On Wii or Wii U vWii**
Wii

**Screenshots**

![IMG_20240715_160653](https://github.com/user-attachments/assets/129da911-bed4-4652-991c-37d01964a9a5)
![IMG_20240715_160640](https://github.com/user-attachments/assets/40659d7b-6eb1-444d-b8eb-0b1e41bf8beb)
![IMG_20240715_160631](https://github.com/user-attachments/assets/caf07f17-d7fc-4dfd-903c-f84081782f0a)
![IMG_20240715_160622](https://github.com/user-attachments/assets/3097e04f-9acd-4154-b3f6-c4e50b42fb6a)
![IMG_20240715_160612](https://github.com/user-attachments/assets/ed28ce36-2727-4cfb-82e3-01519d7dea4e)

**Additional context**

Reloading cache didn't help

When i redownloaded the images, it fixed the covers. But, if you see a cover from the side, it stays corrupt. Selecting it fixes it.

I didn't have these issues before launching AC

**Comment by eku** [comment]
How large is you games / covers collection? We might run out of memory.

**Comment by TheMispeled** [comment]
I have 13 games, wich are 27,4 Go (29 444 014 080 octets) in total.

**Comment by TheMispeled** [comment]
okay, now windows tells me there is a problem with the sd card. I repair it. Now there is a 1,94 gig folder named "FOUND.000" with 8101 files named "FILExxxx.CHK". The wbfs folder is now 24,6 gigs. There is something VERY wrong with this thing.

 I'm vas right since the beginning. The wii is cursed. The sd card is cursed. The old weird thomson 19e72nh00 tv i've used for it is cursed. Everything that interacted with the wii is cursed. Everything is cursed. 

I'm NOT touching this wii again until all the things get fixed.

Also if you couldn't tell, the 2nd paragraph is a joke.

**Comment by TheMispeled** [comment]
It can't be taht the sd card is corrupting, i bought it brand new

**Comment by TheMispeled** [comment]
Ok, so i remade ALL the wbfs files from scratch. Then i reloaded the cache. The images are still corrupted, it takes a bit of moving around to make the corruption disappear. I'm not sure if i should retry to launch AC.

**Comment by gingerbeardman** [comment]
it could be that the filesystem is corrupted on the SD card, rather than the SD card itself. is it a verified reputable brand that came in retail packaging? how is it formatted?

**Comment by TheMispeled** [comment]
It's a SanDisk Extreme Plus, formatted it fat32. But it's most likely that this is not the cause of the issue, because all of this happened after i lauched AC. But yes, it could be this, about the problematic sd card.

**Comment by TheMispeled** [comment]
i dont know what loader to use now. gx freezes for the smallest inconveniencies it encounters. lite shats itself in the most brutal way when someting's wrong.

**Comment by gingerbeardman** [comment]
Did you try a different SD card?

Also, please be respectful in here. Thanks

**Comment by TheMispeled** [comment]
Nope, don't have another SD card the same size. I could try to launch ac on my other SD card.

**Comment by gingerbeardman** [comment]
Please do try it and report back.

Also please confirm exact formatting of both cards and whether or not they pass a disk check test.

**Comment by TheMispeled** [comment]
What should I do? Copy all the games on the other SD card? Only copy animal crossing?

**Comment by gingerbeardman** [comment]
I would copy your base apps setup, with wiiflow, and only animal crossing.

then try it and see
